core/tee/tee_svc.c

```diff
@@ -43,6 +42,12 @@
 #include <trace.h>
 #include <kernel/trace_ta.h>
 #include <kernel/chip_services.h>
+#include <kernel/misc.h>
+
+#define MAX_BASE64_BUFFER_SIZE 120
+
+extern int base64_encode(const unsigned char *in,  unsigned long len,
```

Comment:
I think we need base64_encode() function for kernel space program, but currently we only can have this function in libtomcrypto which can be used by kernel space program. Not sure if there is a better way to use the base64_encode() function instead of declaring it as an extern function?